webpack/bitgojs.config.js

```diff
@@ -42,10 +42,17 @@ module.exports = {
       Buffer: ['buffer', 'Buffer'],
       process: 'process/browser',
     }),
+
+    // new webpack.NormalModuleReplacementPlugin(/\@emurgo\/cardano-serialization-lib-nodejs/, '@emurgo/cardano-serialization-lib-browser'),
```

Comment:
Can we remove the unused code?

Reply:
Sure! Not sure if it matters, but I think we're planning on adding it back once a WASM capability PR is merged by security. But happy to remove completely!